packages/bruno-app/src/utils/codemirror/brunoVarInfo.js

```diff
@@ -6,6 +6,11 @@
  *  LICENSE file at https://github.com/graphql/codemirror-graphql/tree/v0.8.3
  */
 
+// Todo: Fix this
```

Comment:
What about this Todo ?

Reply:
is know bug on import bruno/common, i just copied from other file

Reply:
Ok 
I guess it's fine 😄